packages/material-ui-lab/package.json

```diff
@@ -33,15 +33,15 @@
     "version": "yarn build:copy-files && npm publish build"
   },
   "peerDependencies": {
-    "material-ui": "^1.0.0-beta.33",
```

Comment:
Why removing the peer dependency?

Reply:
It isn't removed - it seems `yarn publish` changes the sort order.

Reply:
Oh, my bad!